box-share-sdk/src/main/java/com/box/androidsdk/share/views/CollaboratorsInitialsView.java

```diff
@@ -146,6 +151,10 @@ public void run() {
                             mProgressBar.setVisibility(GONE);
                             if (response.isSuccess() && getFolder() != null) {
                                 updateView(response.getResult());
+                            } else if (((BoxException)response.getException()).getResponseCode() == 404 ) {
```

Comment:
For these can you use HttpStatus.SC_NOT_FOUND? It's a little more readable then ints.